modulefiles/build_odin_intel

```diff
@@ -0,0 +1,67 @@
+#%Module
+
+proc ModulesHelp { } {
+   puts stderr "This module loads libraries for building SRW on"
+   puts stderr "the NOAA RDHPC machine odin using Intel"
+}
+
+module-whatis "Loads libraries needed for building SRW on odin"
+
+if { [module-info mode load] } {
+  module unload modules
+  system "unset -f module;"
+}
+
+setenv BASH_ENV "/usr/local/lmod/8.3.1/init/bash"
+puts "source $env(BASH_ENV);"
+setenv LMOD_SYSTEM_DEFAULT_MODULES "PrgEnv-intel:cray-mpich:intel:craype"
+module --initial_load --no_redirect restore
+#module use <$HOME>/<your-modulefiles-dir>
+setenv MODULEPATH "/oldscratch/ywang/external/hpc-stack/modulefiles/mpi/intel/2020/cray-mpich/7.7.16:/oldscratch/ywang/external/hpc-stack/modulefiles/compiler/intel/2020:/oldscratch/ywang/external/hpc-stack/modulefiles/core:/oldscratch/ywang/external/hpc-stack/modulefiles/stack:/opt/cray/pe/perftools/21.02.0/modulefiles:/opt/cray/ari/modulefiles:/opt/cray/pe/craype-targets/default/modulefiles:/opt/cray/pe/modulefiles:/opt/cray/modulefiles:/opt/modulefiles"
+
+#module purge
+setenv CMAKE "/home/yunheng.wang/tools/cmake-3.23.0-rc2/bin/cmake"
+setenv PATH "/home/yunheng.wang/tools/cmake-3.23.0-rc2/bin:${PATH}"
+
+module load hpc/1.2.0
+module load hpc-intel
+module load hpc-cray-mpich
+
+#module load srw_common
+
+module load jasper
+module load zlib
+module load png
+
+#module load cray-hdf5
+#module load cray-netcdf
```

Comment:
I assume @ywangwof will be able to test whether these commented lines should/should not be needed here.